modules/ding_place2book/ding_place2book.module

```diff
@@ -1067,52 +1065,68 @@ function ding_place2book_render_place2book_ticketsinfo($node) {
       $p2b_result = drupal_http_request($service_settings['service_url'] . '/available_tickets', $options);
 
       //we must have a sales-status header
-      if ($p2b_result->headers['sales-status']) {
+      $status_type = _ding_place2book_status_processing($p2b_result);
+    }
+  }
+  return theme('place2book_ticketsinfo', array(
```

Comment:
Would it be possible to return an render array here? Would allow easy changes in the theme layer, if any libraries should want to?

Reply:
Yes we can move this into a renderable array, but in this case we will have to review the modality of displaying this button on the frontend and thorough testing. Should we proceed with this?

Reply:
Okay, not a the pressen time as all tests for this release is currently halted and we need to get this fix into the code base.